<launch>
	<!-- arg setting -->
	<arg name="vdSpeed" default="4.5"/>
	<arg name="vdAccel" default="4"/>
	<arg name="vdScrubRad" default=".0762"/>
	
	<!-- node launch shit -->
	<node pkg="mdart" type="vehicle_dynamics" name="vehicle_dynamics">
		<param name="speedLimit" value="$(arg vdSpeed)"/>
		<param name="yAccelLimit" value="$(arg vdAccel)"/>
		<param name="scrubRadius" value="$(arg vdScrubRad)"/>
	</node>
</launch>
